lib/bibliotheca/bot.rb

```diff
@@ -39,6 +39,14 @@ def authenticated?
     !!token
   end
 
+  def upload_image!(slug, filename, content)
+    path = "images/#{filename.gsub(/\.(.*){2,4}/) { |it| "_#{(Time.now.to_f * 1000).to_i}#{it}" }}"
```

Comment:
:scream: 

Perhaps we could extract a method here and test it.

Reply:
Perhaps the bot has no tests...

Reply:
Perhaps you are right